lib/Elevate/OS/CentOS7.pm

```diff
@@ -24,6 +24,8 @@ use constant available_upgrade_paths => (
 use constant default_upgrade_to => 'AlmaLinux';
 use constant ea_alias           => 'CentOS_8';
 use constant elevate_rpm_url    => 'https://repo.almalinux.org/elevate/elevate-release-latest-el7.noarch.rpm';
+use constant leapp_repo_prod    => 'elevate';
+use constant leapp_repo_beta    => '';
```

Comment:
:( almalinux doesn't provide their beta repos in their repo file. Maybe we can get them to add it!

Reply:
CONSIDER:  Move line 28 to `Elevate::OS::RHEL` as a fallback instead of defining it here

Reply:
Good idea! done.